vocabularies.md

```diff
@@ -0,0 +1,282 @@
+        "id": "common_types",
+        "name": "common_types",
+        "scrollable": true,
+        "hierarchical": false,
+        "type": "vocabulary",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7.4science.cloud/server/api/integration/vocabularies/common_types/entries"
+          },
+          "self": {
+            "href": "https://dspace7.4science.cloud/server/api/integration/vocabularies/common_types"
+          }
+        }
+      },
+      {
+        "id": "common_iso_languages",
+        "name": "common_iso_languages",
+        "scrollable": true,
+        "hierarchical": false,
+        "type": "vocabulary",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7.4science.cloud/server/api/integration/vocabularies/common_iso_languages/entries"
+          },
+          "self": {
+            "href": "https://dspace7.4science.cloud/server/api/integration/vocabularies/common_iso_languages"
+          }
+        }
+      }
+    ]
+  },
+  "_links": {
+    "self": {
+      "href": "https://dspace7.4science.cloud/server/api/integration/vocabularies"
+    }
+  },
+  "page": {
+    "size": 20,
+    "totalElements": 3,
+    "totalPages": 1,
+    "number": 0
+  }
+}
+```
+
+## Single Vocabulary
+**/api/integration/vocabularies/<:vocabulary-name>**
+
+Provide detailed information about a specific controlled vocabulary. The JSON response document is as follow
+```json
+{
+  "id": "srsc",
+  "name": "srsc",
+  "scrollable": false,
+  "hierarchical": true,
+  "preloadLevel": 1,
+  "type": "vocabulary"
+}
+```
+
+Attributes
+* id: the id of the vocabulary, it is the same than the name
+* name: see id
+* scrollable: if true mean that it is possible to scroll all the entries in the authority without providing a query parameter, see (vocabulary entries)[vocabularies.md#vocabulary-entries]
+* hierarchical: if true means that the vocabulary expose a tree structure where some entries are parent of others
+* preloadLevel: for hierarchical vocabularies express the preference to preload the tree at a specific level of depth (0 only the top nodes are shown, 1 also their childrens are preloaded and so on)
+
+Exposed links:
+* entries: get entries from the controlled vocabulary
+
+## Linked entities
+### controlled vocabulary entries
+**/api/integration/vocabularies/<:vocabulary-name>/entries**
+
+It returns the entries suggested by the vocabulary in response to the user query or a scrollable list, see below 
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* metadata: the metadata field for which the authority is used: mandatory
+* collection: the uuid of the collection where the item belong to: mandatory
+* query: the terms, keywords or prefix to search
+* exact: can be true or false (default if absent). If true force the vocabulary to provide only entries that match exactly with the query
+* entryID: get the suggestions related to a specific vocabulary entry 
```

Comment:
it could return multiple results, in our slack discussion it was the example related to  http://vocab.getty.edu/page/tgn/6000219 where two entries are expected for the same entryID (Alwa and Alodia)

Reply:
@abollini : Ok, that makes sense. I think we may want to describe this better then.  Maybe say something like:

`entryID:  get just those entries related to a specific vocabulary entry ID.  Multiple entries may be returned, e.g. if two entries are just different terms/phrases for the same thing.`